packages/db/src/migrations/20240423231543_add_github_id.ts

```diff
@@ -0,0 +1,19 @@
+import { type Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  await db.schema
+    .alterTable('students')
+    .addColumn('github_access_token', 'text')
+    .addColumn('github_connected_at', 'timestamptz')
```

Comment:
I added these 2 additional columns so that we can know when the member connected the GitHub account (helpful for tracking adoption over time) as well as the access token itself. That will allow us to actually query things on their GitHub account in the future (ie: commits to Oyster).